From 38615d824e26ebd9f79c8a5a14e1c927de01a783 Mon Sep 17 00:00:00 2001
From: lancethepants <lancethepants@gmail.com>
Date: Wed, 24 Aug 2022 11:11:34 -0600
Subject: [PATCH] package/musl-fts: always build static library

musl-fts provides features left out of musl that are included in other libc's.

This patch always enables building the static library so
that it is always present with musl's libc.a when enabled.

Signed-off-by: Lance Fredrickson <lancethepants@gmail.com>
---
 package/musl-fts/musl-fts.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/musl-fts/musl-fts.mk b/package/musl-fts/musl-fts.mk
index dd7f1f9ea0..fcad9c3d75 100644
--- a/package/musl-fts/musl-fts.mk
+++ b/package/musl-fts/musl-fts.mk
@@ -12,5 +12,6 @@ MUSL_FTS_LICENSE_FILES = COPYING
 # pkg-config needed for autoreconf
 MUSL_FTS_DEPENDENCIES = host-pkgconf
 MUSL_FTS_INSTALL_STAGING = YES
+MUSL_FTS_CONF_OPTS += --enable-static
 
 $(eval $(autotools-package))
-- 
2.20.1

